desktop/src/types.ts

```diff
@@ -206,7 +206,11 @@ export type TImportWorkspaceConfig = Readonly<{
 //#region Context
 export type TContextOptions = Record<TContextOptionName, TContextOption>
 // See pkg/config/context.go
-export type TContextOptionName = "AGENT_URL" | "TELEMETRY"
+export type TContextOptionName =
+  | "AGENT_URL"
+  | "TELEMETRY"
```

Comment:
should these be consts defined somewhere? I see `SSH_INJECT_GIT_CREDENTIALS` references elsewhere as a string